frontend/components/Dashboard/PostCard/postCardUtils.ts

```diff
@@ -0,0 +1,43 @@
+import { format } from 'date-fns'
+import { sanitize } from '../../../utils'
+import { User, PostStatus, Thread } from '../../../generated/graphql'
+
+// TODO: this will be unnecessary once we use types from generated graphql
+export type Post = {
+  id: number
+  title: string
+  body: string
+  readTime: number
+  excerpt: string
+  images?: string[]
+  author: User
+  status: PostStatus
+  likes?: number
+  threads?: Thread[]
+  createdAt: string
+}
+
+export const postBorderRadius = '5px'
+
+export const processPost = (post: Post) => {
```

Comment:
Since we're creating separate components for posts that appear in various pages, this function helps consume the API response of a post and turn it into something that every Post component can use

Reply:
I'm not sure if I fully understand the need for this. Can you explain a bit more the problem we're solving?